tpgtools/overrides/cloudresourcemanager/project.yaml

```diff
@@ -0,0 +1,8 @@
+- type: SERIALIZATION_ONLY
+- type: CUSTOM_SERIALIZER
```

Comment:
Was this added because of a deficiency in the current serialization implementation that there wasn't enough time to incorporate in?
Or is it just a one-off?

Reply:
The CUSTOM_SERIALIZER override is used here because the DCL schema for the project resource differs from the terraform schema.